packages/keyring-api/src/superstruct.ts

```diff
@@ -14,6 +14,7 @@ import type {
   Simplify,
 } from '@metamask/superstruct';
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Same here regarding unused vars

Reply:
Same remark here, it's only used "statically" (for the `type ExactOptionalTag`, so we need to keep it